editor/src/components/editor/store/editor-state.ts

```diff
@@ -2104,12 +2107,62 @@ export function regularNavigatorEntriesEqual(
   return EP.pathsEqual(first.elementPath, second.elementPath)
 }
 
-export interface ConditionalClauseNavigatorEntry {
-  type: 'CONDITIONAL_CLAUSE'
-  elementPath: ElementPath
+export interface ConditionalClause<P extends ElementPath> {
```

Comment:
Why do we need `P extends ElementPath` here?

Reply:
Oh is this needed because of dynamic and static element paths?

Reply:
So that we can have static ones as well as regular ones.